code/game/objects/items/weapons/weaponry.dm

```diff
@@ -66,6 +68,13 @@
 		user << "\blue You hit the floor with the [src]."
 		call(/obj/effect/rune/proc/revealrunes)(src)
 
+/obj/item/weapon/nullrod/pickup(mob/living/user as mob)
+	if(user.mind && user.mind.assigned_role == "Chaplain")
+		user << "<span class ='notice'>The obsidian rod is teeming with divine power. You feel like you could pulverize a horde of undead with this.</span>"
+	if(user.mind && user.mind.vampire && (!VAMP_UNDYING in user.mind.vampire.powers))

```

Comment:
Shouldn't the ! be around the in, not next to VAMP_UNDYING?
That is, !(VAMP_UNDYING in user.mind.vampire.powers)